sdk/keyvault/azure-security-keyvault-secrets/test/samples/sample1-basic-operations/sample1_basic_operations.cpp

```diff
@@ -21,32 +21,44 @@ using namespace std::chrono_literals;
 
 int main()
 {
+  // @begin_snippet: SecretSample1CreateCredential
+  auto tenantId = std::getenv("AZURE_TENANT_ID");
+  auto clientId = std::getenv("AZURE_CLIENT_ID");
+  auto clientSecret = std::getenv("AZURE_CLIENT_SECRET");
```

Comment:
Ah, it looks like I did not quite clean up all the instances/artifacts of ClientSecretCredential. Maybe because this sample file lives in `/test/`.
I'll wait until you merge, and then I'll create a subsequent PR to update, so that we don't collide modifying the same lines.

Reply:
Nah, I'll fix it.

Reply:
Thank you!